From dab93c0d244bbac5c91e6a7b0206d4329fe3624e Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Tue, 15 Sep 2015 12:53:57 +0800
Subject: [PATCH 0630/5242] MLK-11550-2: ARM: dts: imx7d-12x12-lpddr3-arm2:
 add 'fsl,lpsr-mode' for pfuze3000

commit  3c35994d0ad74c9f19a4a6d259ccaef7dcabdf3a from
https://source.codeaurora.org/external/imx/linux-imx.git

Add 'fsl,lpsr-mode' property to support LPSR mode on arm2 board.

Signed-off-by: Robin Gong <b38343@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index da81d59..c8f4482 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -237,6 +237,7 @@
 	pmic: pfuze3000@08 {
 		compatible = "fsl,pfuze3000";
 		reg = <0x08>;
+		fsl,lpsr-mode;
 
 		regulators {
 			sw1a_reg: sw1a {
-- 
1.7.9.5

